hooks/pre/30-upgrade.rb

```diff
@@ -19,6 +19,10 @@ def migrate_candlepin
   Kafo::Helpers.execute("/usr/share/candlepin/cpdb --update --password #{Kafo::Helpers.read_cache_data('candlepin_db_password')}")
 end
 
+def start_tomcat
+  Kafo::Helpers.execute('katello-service start --only tomcat')
```

Comment:
But the script ignores non-existent services, so you can say `--only tomcat,tomcat6`